pkg/3scale/amp/operator/system_reconciler.go

```diff
@@ -28,14 +28,16 @@ func NewSystemReconciler(baseAPIManagerLogicReconciler *BaseAPIManagerLogicRecon
 }
 
 func (r *SystemReconciler) reconcileFileStorage(system *component.System) error {
+	if r.apiManager.IsSystemPostgreSQLEnabled() {
```

Comment:
I think this is wrong. Here we should be checking whether S3 is being used or not. PostgreSQL check (system-database) doesn't seem to be correct here

Reply:
good catch